Users/Tutorial.rst

```diff
@@ -331,7 +331,7 @@ work for all bears.
 ::
 
     # noqa
-    long_line = "This is a long line ... "       
```

Comment:
same here, totally unrelated, probably your editor edited this by mistake and you added this file by mistake